### ISSUE TITLE
screen reader accessibility issue

### ISSUE THREAD

**Issue by bhanuponguru** [mortbopet/Ripes#369]
hello, so i am a screen reader user. to be more specific, i am using windows with NVDA screen reader.
I dont find editor tab, memory tab etc using the screen reader. also the window is not accessible completely, i think it is a qt5 issue but there are other qt5 apps which are maximum accessible. you can look at oracle vm virtual box, it was not accessible before but the recent versions became usable.
here are few suggestions
if it is not possible to update the UI, please add keyboard shortcuts to switch from tab to tab, like editor, memory etc.
instruction memory table, registers table is accessible enough to be usable with NVDA.
rest of the program, almost is not accessible to usable extinct. please let me know if you need any suggestions regarding screen reader accessibility.
thanks in advance.
